OpenTelemetry.sln

```diff
@@ -226,6 +226,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "getting-started-prometheus-
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "customizing-the-sdk", "docs\logs\customizing-the-sdk\customizing-the-sdk.csproj", "{6C7A1595-36D6-4229-BBB5-5A6B5791791D}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "OpenTelemetry.Instrumentation.B3TraceContext", "src\OpenTelemetry.Instrumentation.B3TraceContext\OpenTelemetry.Instrumentation.B3TraceContext.csproj", "{E91B2E40-E428-43B3-8A43-09709F0E69E4}"
```

Comment:
We need to consider how to name this package. I don't think `...Instrumentation...` is what we want.

Looking at some other languages implementations for inspiration:

* opentelemetry-java has placed both the B3 and Jaeger propagators into the same package. https://github.com/open-telemetry/opentelemetry-java/tree/main/extensions/trace-propagators
* opentelemetry-go has done similar, but in addition their package contains an OpenCensus, OpenTracing, and AWS propagator. Furthermore, they maintain this package in their contrib repo. https://github.com/open-telemetry/opentelemetry-go-contrib/tree/main/propagators

I think it makes sense for us to have one package with both the B3 and Jaeger propagators. I propose the name `OpenTelemetry.Extensions.Propagators`. I think it's ok to host it in the opentelemetry-dotnet repo, but open to others' opinions if they think it makes more sense in contrib.